plugins/cookie_manager/pubspec.yaml

```diff
@@ -9,9 +9,9 @@ environment:
   sdk: '>=2.15.0 <3.0.0'
 
 dependencies:
-  cookie_jar: ^3.0.0
-  dio: ^5.0.0
+  cookie_jar: ^4.0.1
+  dio: ^5.1.1
```

Comment:
For major version updates, it makes sense.

Please add a changelog entry which mentions this and bump the version number.
.

Reply:
This is still not done